frontend/src/js/query-runner/DownloadResultsDropdownButton.tsx

```diff
@@ -70,6 +70,10 @@ function getResultUrl(
   );
 }
 
+function truncate(label: string) {
+  return label.length > 40 ? `${label.slice(0, 37)}...` : label;
```

Comment:
Ist geschmackssache, aber es gibt genau dafür das Symbol `…` vlt mag das die Font aber nicht
```suggestion
  return label.length > 40 ? `${label.slice(0, 37)}…` : label;
```